mne/viz/ica.py

```diff
@@ -329,17 +356,50 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
 
     # calculations
     # ------------
+
     if isinstance(inst, BaseRaw):
+        # if reject is set, reject
+        if reject == 'auto':
+            reject = getattr(ica, 'reject_', None)
+        if reject is not None:
+            data = inst.get_data()
+            data, drop_inds = _reject_data_segments(data, ica.reject_,
+                                                    flat=None, decim=None,
+                                                    info=inst.info,
+                                                    tstep=2.0)
+            inst_rejected = RawArray(data, inst.info)
+        else:
+            inst_rejected = inst
+            drop_inds = None
         # break up continuous signal into segments
         from ..epochs import _segment_raw
+        inst_rejected = _segment_raw(inst_rejected, segment_length=2.,
+                                     verbose=False,
+                                     preload=True)
         inst = _segment_raw(inst, segment_length=2., verbose=False,
                             preload=True)
         kind = "Segment"
     else:
+        drop_inds = None
+        inst_rejected = inst
         kind = "Epochs"
 
-    epochs_src = ica.get_sources(inst)
-    ica_data = np.swapaxes(epochs_src.get_data()[:, picks, :], 0, 1)
+    epochs_src = ica.get_sources(inst_rejected)
+    data = epochs_src.get_data()
+
+    ica_data = np.swapaxes(data[:, picks, :], 0, 1)
+
+    # getting dropped epochs indexes
+
+    dropped_indices = []
+    if drop_inds is not None:
+        for dropped in drop_inds:
+            dropped_indices.append((dropped[0] // len(inst.times)) + 1)
+
+    # getting
```

Comment:
I don't understand the comment. I would remove it. But even if you let it, remove the empty line after the comment.